R/query.R

```diff
@@ -53,7 +53,7 @@
 #' docdb_query(src, "iris", query = "Petal.Width:1.5")
 #'
 #' # Mongo
-#' src <- src_mongo()
```

Comment:
also fails:

```r
src <- src_mongo(collection = "mtcars")
if (docdb_exists(src, "mtcars")) docdb_delete(src, "mtcars")
docdb_create(src, "mtcars", mtcars)
docdb_query(src, query = '{"mpg":21}')
#> Error in docdb_query.src_mongo(src, query = "{\"mpg\":21}") :
#>   argument "key" is missing, with no default
docdb_query(src, query = '{"mpg":21}', fields = '{"mpg":1, "cyl":1}')
#> Error in docdb_query.src_mongo(src, query = "{\"mpg\":21}", fields = "{\"mpg\":1, \"cyl\":1}") :
#>   argument "key" is missing, with no default
```

Reply:
Thanks, completed and corrected examples in documentation with commit 19b6d9b.

Reply:
LGTM